Task.js

```diff
@@ -102,6 +108,51 @@ class Task extends Unit {
     );
   }
 
+  isURL(source) {
+    try {
+      new URL(source);
+      return true;
+    } catch (e) {
+      return false;
+    }
+  }
+
+  jsPath() {
+    let final_paths = [];
+    for (let js_path of this.js_modules) {
+      if (this.isURL(js_path)) {
+        final_paths.push(js_path);
+        continue;
+      }
+
+      const absolute_path = path.resolve(
+        path.join(Config.build_path(this.exp_path), this.basedir, js_path)

```

Comment:
What happens if the path.resolve returns an error, in case the path mentioned is not a valid path in the system?

Reply:
path.resolve will return a relative path to that file, after that, we use fs.existsSync to check whether the file actually exists or not.